### PR TITLE
Added TIX ID, AnTuTu 3DBench and AOSPStudio's Files Shortcut

### PR DESCRIPTION
**Pull request by ZhmieXD**
Sorry, a small one this time, mainly because there are some apps I just installed but haven't got themed and it kinda bugs me out, but don't worry, I'll add more in the future 👍 

# Added and Linked Icons

1. Antutu 3DBench (`com.antutu.benchmark.full`) -> Linked to `@drawable/antutu`
2. AOSPStudio's Files Shortcut (`org.aospstudio.files`) -> Linked to `@drawable/files`
3. TIX ID (`id.tix.android`) -> New

Preview for `tix_id.svg:`

image

#

Thanks beforehand! 😁

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -173,13 +174,14 @@
...
     <icon drawable="@drawable/fdroid" package="org.fdroid.fdroid" name="F-Droid" />
     <icon drawable="@drawable/files" package="com.android.documentsui" name="Files" />
+    <icon drawable="@drawable/files" package="com.android.filemanager" name="Files" />
     <icon drawable="@drawable/files" package="com.coloros.files" name="Files" />
     <icon drawable="@drawable/files" package="com.google.android.documentsui" name="Files" />
     <icon drawable="@drawable/files" package="com.huawei.hidisk" name="Files" />
     <icon drawable="@drawable/files" package="com.oneplus.filemanager" name="Files" />
     <icon drawable="@drawable/files" package="com.sec.android.app.myfiles" name="Files" />
     <icon drawable="@drawable/files" package="me.zhanghai.android.files" name="Files" />
-    <icon drawable="@drawable/files" package="com.android.filemanager" name="File Manager" />
```
Comment (lorrding) [inline app/src/main/res/xml/grayscale_icon_map.xml:182]: Seems like a mistake?

Comment (ZhmieXD) [reply app/src/main/res/xml/grayscale_icon_map.xml:182]: I renamed that on purpose to avoid redundancy on the Lawnicons app itself

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Thanks for the PR, and no need to apologize for the number of changes! All contributions are highly appreciated. Iʼd suggest these minor geometric tweaks: [`tix_id.zip`](https://github.com/LawnchairLauncher/lawnicons/files/9126407/tix_id.zip).

**Comment by ZhmieXD** [comment]
Umm, okay I just applied the suggested changes but looks like I f'ed up my remote that I can't push it ^^;

Will try to refork and resubmit the pull request with the suggested changes. Sorry for the inconvenience ^^;

**Comment by patrykmichalik** [comment]
It’s all good! Feel free to open a new PR.

**Pull request closed without merging**